index.bs

```diff
@@ -1264,9 +1266,18 @@ Reflective modules are always already instantiated.
 
 <emu-alg>
 1. Let _module_ be this Reflective Module Record.
-1. Let _evaluate_ be _module_.[[Evaluate]].
+1. If _module_.[[Evaluated]] is *true*, return *undefined*.
+1. Let _func_ be _module_.[[Evaluate]].
+1. Set _module_.[[Evaluated]] to *true*.
 1. Set _module_.[[Evaluate]] to *undefined*.
-1. Return _evaluate_().
+1. Assert: _func_ is callable.
+1. For each _dep_ in entry.[[Dependencies]], do:
```

Comment:
A reflective module might depends on another module that requires evaluation because you can pass a namespace object via re-export descriptor, and we need to guarantee the execution of that piece as well.